process/transaction/shardProcess.go

```diff
@@ -827,8 +842,6 @@ func (txProc *txProcessor) executeFailedRelayedTransaction(
 		if err != nil {
 			return err
 		}
-
-		return process.ErrFailedTransaction
```

Comment:
This removal was the second problem that I found yesterday but I forgot to mention about it :+1: